### DISCUSSION TITLE
Extension Not working after setup

### DISCUSSION THREAD

**Discussion by BejadiRajeshReddy** [open-sauced-craftwork/browser-extension#15] [Q&A] [answered]
**Everything is clear but Extension is not working**

![Screenshot (441)](https://user-images.githubusercontent.com/64033035/224682261-e2338aa9-46b1-4b37-8a83-28beefd88192.png)
![Screenshot (442)](https://user-images.githubusercontent.com/64033035/224682274-f6d56061-e459-4826-89f1-ed8779c2077a.png)

**Comment by hitenvats16** [comment, marked as answer]
Went through same issue.
I just imported React in App.jsx and it worked fine for me.

P.S: Whenever you're stuck with en error, always check console first because most of the time errors themselves convey how to fix them. You can check console for the extension by right clicking on extension and selecting "Inspect Pop-up"
